src/XamlX/Transform/Transformers/XamlIntrinsicsTransformer.cs

```diff
@@ -87,6 +91,49 @@ XamlAstTextNode ResolveArgumentOrValue(string extension, string name)
                         new XamlAstXmlTypeReference(ni, resolvedNs, tmpair[0], xml.GenericArguments),
                         tmpair[1]);
                 }
+
+                if (xml.Name == "Array")
+                {
+                    var elements = new List<IXamlAstValueNode>();
+                    elements.AddRange(ni.Arguments);
+
+                    IXamlAstValueNode type = null;
+
+                    foreach (var child in ni.Children)
+                    {
+                        if (child is XamlAstXamlPropertyValueNode propertyNode)

```

Comment:
We need to trigger an error for unhandled nodes, otherwise they will be silently ignored.